libkbfs/kbfs_ops_test.go

```diff
@@ -1031,7 +1018,7 @@ func TestKBFSOpsLookupNoSuchNameFail(t *testing.T) {
 	}
 }
 
-func TestKBFSOpsLookupNewDataVersionFail(t *testing.T) {
+func TestKBFSOpsReadNewDataVersionFail(t *testing.T) {
```

Comment:
Why this change?

Reply:
You removed the `checkDataVersion` call that used to be in `Lookup`, so this test doesn't work anymore when the parent block is served from the cache.

Reply:
Makes sense, thanks.